gccapp/src/components/squares/index-home.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { SquareContainer, SquareItem, ImageRow, TextRow, SquareTitle, SquareText, BooksImage, PathImage, PicsImage } from './elements';
+import books from '../../assets/homepage/books.png';
+import path from '../../assets/homepage/path.png';
+import pics from '../../assets/homepage/pics.png';
+
+const SquaresHome = () => {
+    const handleClick = (path : string) => {
+        window.location.href = path;
+    };
+
+    return (
+        <SquareContainer>
+            <SquareItem onClick={() => handleClick('/journey')}>
+                <ImageRow>
+                    <PathImage>
+                        <img src={path} alt="path" />
+                    </PathImage>
+                </ImageRow>
+                <TextRow>
+                    <SquareTitle>E-learning Journey</SquareTitle>
+                    <SquareText>To-do Content</SquareText>
+                </TextRow>
+            </SquareItem>
+            <SquareItem onClick={() => handleClick('/about')}>
+                <ImageRow>
+                    <BooksImage>
+                        <img src={books} alt="books" />
+                    </BooksImage>
+                </ImageRow>
+                <TextRow>
+                    <SquareTitle>About</SquareTitle>
+                    <SquareText>Reading Content</SquareText>
+                </TextRow>
+            </SquareItem>
+            <SquareItem onClick={() => handleClick('/badges')}>
+                <ImageRow>
+                    <PicsImage>
+                        <img src={pics} alt="pics" />
+                    </PicsImage>
+                </ImageRow>
+                <TextRow>
+                    <SquareTitle>Badge Collection</SquareTitle>
+                    <SquareText>To-do Content</SquareText>
+                </TextRow>
+            </SquareItem>
+        </SquareContainer>
+    );
+}
+
+export default SquaresHome;
+
+/*
+import {SquareContainer, SquareItem, ImageRow, TextRow, SquareTitle, SquareText, BooksImage, PathImage, PicsImage} from './elements';
```

Comment:
Do we need this?

Reply:
I'll keep it in for now, we can clean it up later (aka probably lives in the codebase forever lol)